exporter/elasticexporter/go.mod

```diff
@@ -0,0 +1,18 @@
+module github.com/open-telemetry/opentelemetry-collector-contrib/exporter/elasticexporter
+
+go 1.14
+
+require (
+	github.com/elastic/go-sysinfo v1.3.0 // indirect
+	github.com/elastic/go-windows v1.0.1 // indirect
+	github.com/golang/protobuf v1.4.1 // indirect
```

Comment:
This is the likely cause for build failure. It [was reported](https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/230#issuecomment-629353362) to cause performance regression.

Reply:
Ah, thanks! I've tidied go.mod, reverted to v1.3.5.